stdimage/models.py

```diff
@@ -128,7 +144,7 @@ def delete(self, save=True):
         super(StdImageFieldFile, self).delete(save)
 
     def delete_variations(self):
-        for variation in self.field.variations:
+        for _, variation in self.field.variations.items():
```

Comment:
I had to make the `StdImageFieldFile.get_variation_name` method use the full variation dict, because resulting filename now depends on more than just a variation name - the `is_progressive_jpeg` property might change filename extension. I've preserved ability to get filename by just a name, but that is error-prone and should not be used (i.e. filenames generated for progressive variations might have wrong extension).

I guess that ability of getting variation filename by just a variation name should be marked for deprecation.